app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,16 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, bow_level: int, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        """Get player info."""
+        return (f"Elf ranger {self.nickname}. {self.nickname} "
```

Comment:
Use double quotes for consistency in your code. The string should use double quotes as per the checklist.